pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/crd/patches/enablewebhook_patch.go

```diff
@@ -66,5 +66,6 @@ spec:
           namespace: system
           name: webhook-service
           path: /convert
+      conversionReviewVersions: v1
```

Comment:
This is a list type not string. Also I think we can safely assume that CRD version matches conversion review version.
```suggestion
      conversionReviewVersions:
      - {{ .Resource.API.CRDVersion }}
```